test/steps/v2_steps.py

```diff
@@ -1710,8 +1754,12 @@ def split_and_process_app_args(in_args):
     sub_args = [sub_arg.split(":") for sub_arg in split_args]
     app_args = []
     for sub_arg in sub_args:
-        if sub_arg[0] == "str":
+        if len(sub_arg) == 1:  # assume int
```

Comment:
allowing int's in the case when we want to pick up the i'th  app that was stored in the context